### PR TITLE
hai reveal grammar change

### PR DESCRIPTION
**Pull request by zlonghofer**
## Summary
Grammar change to Hai Reveal 4

## Fix Details
May you implies that the militia captain is requesting that the pilot ask permission to hunt the pirates. Will you is a direct request for the pilot to do the bounty hunting mission.

## UI Screenshots
N/A

## Performance Impact
None

### CODE REVIEW

File: data/hai/hai reveal 4 middle.txt [hunk 1]
```diff
@@ -332,7 +332,7 @@ mission "Hai Reveal: Pirate Troubles [1]"
 		"reputation: Hai Merchant (Human)" += 20
 		"reputation: Hai Merchant (Sympathizers)" += 20
 		conversation
-			`The battle has ended, but the spaceport still seems more hectic than usual. Ships are jumping in from the surrounding systems to help any injured and to repair any damage that may have occurred. After some hours pass and the commotion has died down, a Hai in uniform approaches you. "We have heard from several people here that you are considered a seasoned fighter among your people, and learned of your current status assisting our government on other matters. May you lend your combat prowess to us to resolve this issue?"`
+			`The battle has ended, but the spaceport still seems more hectic than usual. Ships are jumping in from the surrounding systems to help any injured and to repair any damage that may have occurred. After some hours pass and the commotion has died down, a Hai in uniform approaches you. "We have heard from several people here that you are considered a seasoned fighter among your people, and learned of your current status assisting our government on other matters. Will you lend your combat prowess to us to resolve this issue?"`
```
Comment (EjoThims) [inline data/hai/hai reveal 4 middle.txt:335]:
```suggestion
			`The battle has ended, but the spaceport still seems more hectic than usual. Ships are jumping in from the surrounding systems to help any injured and to repair any damage that may have occurred. After some hours pass and the commotion has died down, a Hai in uniform approaches you. "We have heard from several people here that you are considered a seasoned fighter among your people and learned of your current status assisting our government on other matters. Will you lend your combat prowess to us to resolve this issue?"`
```

**Comment by EjoThims** [comment]
You're right about the implication, but that may still be appropriate and deliberate, if the intent of the uniformed Hai is to ask if "your current status assisting our government on other matters" allows you to help now with this problem.

@MasterOfGrey 

That extra comma should come out, either way, though.

**Comment by zlonghofer** [comment]
I could see where you're coming from there. In this case, would the uniformed Hai be asking if the pilot has time (is available to help), or asking if the Hai government would allow the pilot to further help other Hai? In the former case, perhaps a different rewording would be clearer; in the latter, I believe "May" could stay.

**Comment by MasterOfGrey** [comment]
I think if we're going to make a change here it should be, "Could you" or "Would you"

File: data/hai/hai reveal 4 middle.txt [hunk 2]
```diff
@@ -332,7 +332,7 @@ mission "Hai Reveal: Pirate Troubles [1]"
 		"reputation: Hai Merchant (Human)" += 20
 		"reputation: Hai Merchant (Sympathizers)" += 20
 		conversation
-			`The battle has ended, but the spaceport still seems more hectic than usual. Ships are jumping in from the surrounding systems to help any injured and to repair any damage that may have occurred. After some hours pass and the commotion has died down, a Hai in uniform approaches you. "We have heard from several people here that you are considered a seasoned fighter among your people, and learned of your current status assisting our government on other matters. May you lend your combat prowess to us to resolve this issue?"`
+			`The battle has ended, but the spaceport still seems more hectic than usual. Ships are jumping in from the surrounding systems to help any injured and to repair any damage that may have occurred. After some hours pass and the commotion has died down, a Hai in uniform approaches you. "We have heard from several people here that you are considered a seasoned fighter among your people and learned of your current status assisting our government on other matters. Will you lend your combat prowess to us to resolve this issue?"`
```
Comment (EjoThims) [inline data/hai/hai reveal 4 middle.txt:335]:
```suggestion
			`The battle has ended, but the spaceport still seems more hectic than usual. Ships are jumping in from the surrounding systems to help any injured and to repair any damage that may have occurred. After some hours pass and the commotion has died down, a Hai in uniform approaches you. "We have heard from several people here that you are considered a seasoned fighter among your people and learned of your current status assisting our government on other matters. Would you lend your combat prowess to us to resolve this issue?"`
```

Gonna suggest this of Grey's recommendations. Bit stronger direct ask than the other.

**Review by EjoThims — CHANGES_REQUESTED** [review]

File: data/hai/hai reveal 4 middle.txt (hunk 2 above)
Comment (UnorderedSigh) [reply data/hai/hai reveal 4 middle.txt:335]: My thoughts:

- May you = grammar error
- Will you = This problem needs to be fixed. Is this something you are going to fix?
- Would you = We need this problem to be fixed. Please fix it for us.
- Could you = Please, kind sir, help us fix this problem, if it isn't too much trouble for you.

Given that you are about the same level as Hai planetary defense officers in the social pecking order, and given that it is a request you really shouldn't refuse, "would you" seems best to me.

**Review by UnorderedSigh — APPROVED** [review]

File: data/hai/hai reveal 4 middle.txt (hunk 2 above)
Comment (Zireael07) [reply data/hai/hai reveal 4 middle.txt]: English major here: "May you" is NOT a grammar error. The meaning is similar to "Could you", however, so "would you" is probably the best

**Pull request merged**